docs/parameters.md

```diff
@@ -25,22 +28,25 @@ Parameters a user can specify when provisioning an FSx OpenZFS file system:
+| `CopyTagsToBackups`             | No        | `'false'`                                                                                                                                                                                                                                                                                |                                                                                                                                                      |
+| `CopyTagsToVolumes`             | No        | `'false'`                                                                                                                                                                                                                                                                                |                                                                                                                                                      |
+| `DailyAutomaticBackupStartTime` | No        | `'"00:00"`                                                                                                                                                                                                                                                                               |                                                                                                                                                      |
+| `DiskIopsConfiguration`         | No        | `'{"Iops": 300, "Mode": "USER_PROVISIONED"}'`                                                                                                                                                                                                                                            |                                                                                                                                                      |
+| `EndpointIpAddressRange`        | No        | `'"192.168.0.0/24"'`                                                                                                                                                                                                                                                                     |                                                                                                                                                      |
+| `PreferredSubnetId`             | MAZ       | `'"subnet-016affca9638a1e61"'`                                                                                                                                                                                                                                                           |                                                                                                                                                      |
```

Comment:
Should probably specify "Required for MULTI_AZ_1" instead of just MAZ

Reply:
Fixed